.vscode/launch.json

```diff
@@ -1,6 +1,57 @@
 {
   "version": "0.2.0",
   "configurations": [
+    {
+      "type": "node",
+      "request": "launch",
+      "name": "Debug - API",
+      "cwd": "${workspaceFolder}/sources/packages/backend",
+      "runtimeExecutable": "npm",
+      "runtimeArgs": ["run", "start:debug", "api"],
+      "autoAttachChildProcesses": true,
+      "restart": true,
+      "sourceMaps": true,
+      "stopOnEntry": false,
+      "console": "integratedTerminal",
+      "sourceMapPathOverrides": {
+        "webpack:///./~/*": "${workspaceFolder}/sources/packages/backend/node_modules/*",
+        "webpack://?:*/*": "${workspaceFolder}/sources/packages/backend/*"
+      }
+    },
+    {
+      "type": "node",
+      "request": "launch",
+      "name": "Debug - Workers",
+      "cwd": "${workspaceFolder}/sources/packages/backend",
+      "runtimeExecutable": "npm",
+      "runtimeArgs": ["run", "start:debug", "workers"],
+      "autoAttachChildProcesses": true,
+      "restart": true,
+      "sourceMaps": true,
+      "stopOnEntry": false,
+      "console": "integratedTerminal",
+      "sourceMapPathOverrides": {
+        "webpack:///./~/*": "${workspaceFolder}/sources/packages/backend/node_modules/*",
+        "webpack://?:*/*": "${workspaceFolder}/sources/packages/backend/*"
+      }
+    },
+    {
+      "type": "node",
+      "request": "launch",
+      "name": "Debug - Queue-consumers",
+      "cwd": "${workspaceFolder}/sources/packages/backend",
+      "runtimeExecutable": "npm",
+      "runtimeArgs": ["run", "start:debug", "queue-consumers"],
+      "autoAttachChildProcesses": true,
+      "restart": true,
+      "sourceMaps": true,
+      "stopOnEntry": false,
+      "console": "integratedTerminal",
+      "sourceMapPathOverrides": {
+        "webpack:///./~/*": "${workspaceFolder}/sources/packages/backend/node_modules/*",
+        "webpack://?:*/*": "${workspaceFolder}/sources/packages/backend/*"
+      }
+    },
     {
```

Comment:
As you are adding debug for these, can we please add the test run for workflow too??

{
      "type": "node",
      "request": "launch",
      "name": "Workflow - Current test file",
      "program": "${workspaceFolder}/sources/packages/backend/node_modules/.bin/jest",
      "args": [
        "${fileBasenameNoExtension}",
        "--runInBand",
        "--config",
        "./sources/packages/backend/workflow/test/jest-e2e.json",
        "--forceExit"
      ],
      "console": "integratedTerminal",
      "internalConsoleOptions": "neverOpen",
      "disableOptimisticBPs": true,
      "windows": {
        "program": "${workspaceFolder}/sources/packages/backend/node_modules/jest/bin/jest"
      },
      "env": {
        "ENVIRONMENT": "test",
        "TZ": "UTC"
      }
    },
    {
      "type": "node",
      "request": "launch",
      "name": "Workflow - all tests",
      "program": "${workspaceFolder}/sources/packages/backend/node_modules/.bin/jest",
      "args": [
        "--runInBand",
        "--config",
        "./sources/packages/backend/workflow/test/jest-e2e.json",
        "--forceExit"
      ],
      "console": "integratedTerminal",
      "internalConsoleOptions": "neverOpen",
      "disableOptimisticBPs": true,
      "windows": {
        "program": "${workspaceFolder}/sources/packages/backend/node_modules/jest/bin/jest"
      },
      "env": {
        "ENVIRONMENT": "test",
        "TZ": "UTC"
      }
    },